skietbaan_fe/src/components/RegisterMemberStyles.css

```diff
@@ -32,15 +28,19 @@
     box-shadow: 2px 2px rgba(0,0,0,0.24);
   }
   @media screen and (min-width:768px)  {
-  .centre-register{
+  .centre-registerMember{
     width: 50%;
```

Comment:
so if you want to center something there are better ways to go about it, using margin-left/margin-right: auto;
(only works if the element has a set width), in some cases text-center might even work on elements and it's contents as well, also..

what is the top 8% suppose to be for?